scipy/optimize/_minimize.py

```diff
@@ -605,7 +605,7 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
         constraints = standardize_constraints(constraints, x0, meth)
 
     if meth == 'nelder-mead':
-        return _minimize_neldermead(fun, x0, args, callback, **options)
+        return _minimize_neldermead(fun, x0, args, callback, bounds, **options)
```

Comment:
Prefer to call as a keyword `bounds=bounds`, but not essential.

Reply:
The reason being that whilst `_minimize_neldermead` is a private function it's better to add new keywords to the end of the function definition.

Reply:
I fixed it.